pkg/scaffold/v2/types.go

```diff
@@ -38,7 +38,7 @@ type Types struct {
 // GetInput implements input.File
 func (t *Types) GetInput() (input.Input, error) {
 	if t.Path == "" {
-		t.Path = filepath.Join("pkg", "apis", t.Resource.Group, t.Resource.Version,
+		t.Path = filepath.Join("api", t.Resource.Version,
```

Comment:
it does not show correctly.

Reply:
This is `v2` where the path was being changed manually in `pkg/scaffold/api` to this value. `v1` keeps his original value.